raft/log_unstable_test.go

```diff
@@ -205,15 +203,9 @@ func TestUnstableRestore(t *testing.T) {
 	s := pb.Snapshot{Metadata: pb.SnapshotMetadata{Index: 6, Term: 2}}
 	u.restore(s)
 
-	if u.offset != s.Metadata.Index+1 {
-		t.Errorf("offset = %d, want %d", u.offset, s.Metadata.Index+1)
-	}
-	if len(u.entries) != 0 {
-		t.Errorf("len = %d, want 0", len(u.entries))
-	}
-	if !reflect.DeepEqual(u.snapshot, &s) {
-		t.Errorf("snap = %v, want %v", u.snapshot, &s)
-	}
+	require.Equal(t, s.Metadata.Index+1, u.offset)
+	require.Zero(t, len(u.entries))
```

Comment:
for next time: `require.Zero(t, u.entries)` is better because it will print the slice if it's not empty.

Reply:
ok!